site/messages/en.json

```
@@ -102,17 +102,17 @@
   "update-funding-rate": "Update Funding Rate",
   "unpause-stream": "Unpause Stream",
   "unwrap-weth-success": "You unwrapped {value} WETH successfully!",
-  "unwrap": "unwrap",
+  "unwrap": "Unwrap Ethereum",
   "utilties": "Utilities",
   "value": "Value",
-  "verify-signature-etherscan": "Verify signature on Etherscan",
-  "view-source-code": "View source code",
+  "verify-signature-etherscan": "Verify on Etherscan",
+  "view-source-code": "Source code",
   "winning-price": "Winning price",
   "won-by": "Won by",
   "won": "Won",
   "wrap-eth-success": "You wrapped {value} ETH successfully!",
   "wrap-unwrap-eth": "Wrap/Unwrap ETH",
-  "wrap": "wrap",
+  "wrap": "Wrap Ethereum",
```

`Ether` (the token) is being wrapped/unwrapped. 

```suggestion
  "wrap": "Wrap Ether",
```

---

Fixed here: https://github.com/hemilabs/pure.finance/pull/2/commits/cacff18224bff82f9fb329376126dffff2239d91